src/executables/util_service.c

```diff
@@ -4672,7 +4632,7 @@ process_heartbeat_start (HA_CONF * ha_conf, int argc, const char **argv)
 	}
     }
 
-  status = us_hb_process_start (ha_conf, db_name, true);
+  status = us_hb_process_start (ha_conf, db_name);
   if (status != NO_ERROR)
```

Comment:
In the error case, us_hb_process_stop() can be called, and us_hb_copylogdb_stop()/us_hb_applylogdb_stop() are called inside. I think they are removed along since the start of them are removed in this PR. Are they gonna removed in another PR?

Reply:
Yes right. they will be removed in next PR. 
After removing all the part that can start the copylogdb and applylogdb, then I will remove them.

Reply:
start and stop are usually a pair. I think it'd be better to remove them at once to not lose track. For example, after this PR is merged, it's hard to know why copylogdb and applyogdb are stopped during the error handling. Anyway, if you'd like to keep separating them, please link each stop where it started, in the next PR. Thanks.

Reply:
> start and stop are usually a pair. I think it'd be better to remove them at once to not lose track.
OK, I agree. I will remove the part that stop the copylogdb/applylogdb.